mozilla/fx-private-relay
# MPP-1515: Hero Image on Premium/Homepage is tranlasted into French/German for French/German users

Summary: 
If a user visits the homepage or Premium interstitial page in French or German, the main hero graphic text is translated into their respective language. 

This PR fixes [MPP-1515](https://mozilla-hub.atlassian.net/browse/MPP-1515)

How to test:
- Set browser language to German or French
- Navigate to the [Premium page](http://127.0.0.1:8000/premium) or the [homepage](http://127.0.0.1:8000/)
- **Expected:** All text, including the hero image, should be in the set language. 

Checklist: 

## Screenshots

French:
![image](https://user-images.githubusercontent.com/2692333/148297008-de80c2a6-dd57-4e6e-a365-ba90f02660f9.png)


German: 
![image](https://user-images.githubusercontent.com/2692333/148296964-0cc4bfca-4652-4e49-912f-f680097bcd02.png)
